### PR TITLE
Update SUMMARY.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Redirects for remaining sections of "how to spend company money". 

@TQuock: The first test I did was successful so I have moved a few more pages over to the new Finance section. However, I suspect the nesting may not be as expected. Would you be open to creating the structure based on your spreadsheet. I'll then fill in any gaps in terms of missing READMEs. There are some similarly-named directories and pages (spending company money) that would be great to resolve as well.

Once this is ironed out I think we'll be able to test what we have and confirm it's all working as expected.

If you're able to work on my branch (https://github.com/mattermost/mattermost-handbook/tree/justine-redirects-phase2) for these changes that'd be awesome so we can keep it all in one place. Please let me know if you have any issues or can't edit the file.

@jasonblais, adding you for visibility and blindspot checks. :)

### CODE REVIEW

File: SUMMARY.md
Comment (jasonblais) [inline SUMMARY.md]: missing a link?

**Review by jasonblais — COMMENTED** [review]
Thanks! One question is whether we want to move/restructure the GitHub files to be in this same structure as well, or keep them where they are? (e.g. do we keep the 'how to spend company money' under 'how to guides for staff' folder in GitHub, or move that file to be in the 'staff member expenses' folder

File: SUMMARY.md
Comment (justinegeffen) [reply SUMMARY.md]: Yeah, I haven't created all the pages which isn't a great user experience. I'll add all the pages first.

**Comment by justinegeffen** [comment]
@TQuock, feel free to make edits to this. If any other pages are moved around please let me know so I can add the redirects if needed. :)

**Comment by justinegeffen** [comment]
Good question, @jasonblais. It makes sense to do it, I'm 5/5 on that. I just need to figure out how the redirect would work. They currently use the original path and then point to the new path. But that's with the file in the original location. I'd like to see whether these redirects work first, and then I'd have a better idea of next steps.

**Review by jasonblais — APPROVED** [review]
Looks good, but let's test the redirects once merged

**Comment by jasonblais** [comment]
@justinegeffen I'll let you merge so you're online to test things out. Thanks for the changes!

**Comment by TQuock** [comment]
@justinegeffen everything looks good when I look at the summary page ToC but when I go to the [handbook](https://handbook.mattermost.com/operations/spending-company-money) it doesn't mirror the summary page

**Review by TQuock — APPROVED** [review]
@justinegeffen  as discussed, we are going to push this PR to get better visibility in the changes. We can then troubleshoot afterwards, if necessary

**Pull request merged**
